lib/helper/WebDriver.js

```diff
@@ -567,6 +567,9 @@ class WebDriver extends Helper {
       });
     }
 
+    if (this.browser.capabilities && this.browser.capabilities.platformName) {
+      this.browser.capabilities.platformName = this.browser.capabilities.platformName.lowercase();
```

Comment:
```suggestion
      this.browser.capabilities.platformName = this.browser.capabilities.platformName.lowercase();
```
maybe it should be https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/toLowerCase ?

Reply:
Sorry, my bad! Updated.